infra/modules/worklytics-connector-specs/main.tf

```diff
@@ -783,4 +783,13 @@ locals {
   enabled_bulk_connectors = {
     for k, v in local.bulk_connectors : k => v if contains(var.enabled_connectors, k)
   }
-}
+
+  enabled_lockable_oauth_secrets_to_create = distinct(flatten([
+  for k, v in local.enabled_oauth_long_access_connectors : [
+  for secret_var in v.secured_variables : {
+    connector_name = k
+    secret_name    = secret_var.name
+  } if try(secret_var.lockable, false) == true
```

Comment:
Exposed the secrets needs to be locked; nothing for AWS but for GCP it will be used to assign the specific role created for